pydy/bodies.py

```diff
@@ -0,0 +1,59 @@
+from sympy.physics.mechanics import RigidBody
+
+from .force import Force
+
+__all__ = ['Body', 'Ground']
+
+
+class Body(RigidBody):
```

Comment:
I intend to create a separate subclass of Particle to incorporate things required to build joints system.